Documentation/release-notes/4567.md

```diff
@@ -0,0 +1,18 @@
+### Preliminary JDK 11 Support
+
+Xamarin.Android can now use JDK 11 to build libraries and applications.
+
+To do so:
+
+ 1. Install JDK 11
+ 2. Set the `$(JavaSdkDirectory)` MSBuild property to refer to the root
+    of the JDK installation from (1).
+
+#### Known Issues
+
+When JDK 11 is used, the `@(JavaDocJar)` Build action is not supported,
+and binding projects will not parse Javadoc to determine parameter names.
+This may a change in parameter names.
```

Comment:
It looks like there's a word or two missing here unintentionally, probably just due to rearrangements in editing. Maybe this can be something like:

```
This may result in different parameter names for existing bindings projects.
Authors of bindings projects affected by this change are recommended to continue
to use JDK 8 to build the bindings projects until this issue is resolved.
```

The additional candidate sentence also helps reinforce the point that this issue is primarily relevant to bindings projects authors and doesn't affect app project authors who are only consuming bindings provided by the Xamarin team.